create_infotexts.py

```diff
@@ -211,40 +210,34 @@ def main():
     # print(places_mapping)
 
     metadata = load_json_metadata(metadata_json)
-    batch_dict = {}
+    batch_info = {}
     for fotonr in metadata:
-        print(fotonr)
-        batch_dict[fotonr] = {}
+        img_info = {}
 
         full_infotext = ""
 
         commons_filename = create_commons_filename(metadata, fotonr)
         #print("New filename: {}".format(commons_filename))
-        batch_dict[fotonr]["filename"] = commons_filename
+        img_info["filename"] = commons_filename
 
         img = CypernImage()
         infobox = generate_infobox_template(metadata[fotonr], img, places_mapping)
-        batch_dict[fotonr]["infotext"] = infobox
+        img_info["info"] = infobox
 
-        content_cats = img.content_cats
-        content_cats_list = []
-        for cat in content_cats:
-            cat_as_wikitext = "[[Category:{cat}]]".format(cat=cat)
-            content_cats_list.append(cat_as_wikitext)
-        content_cats_str = "\n".join(content_cats_list)
-        batch_dict[fotonr]["content_cats"] = content_cats_str
+        img_info["cats"] = img.content_cats
 
         meta_cats = img.meta_cats
```

Comment:
You missed the comment on meta_cats. I.e. you don't need to format them either. just use `img_info["meta_cats"] = img.meta_cats`